src/DssDirectDepositHub.t.sol

```diff
@@ -370,13 +370,13 @@ contract DssDirectDepositHubTest is DSTest {
         _windSystem();
 
         // Set ilk line below current debt
-        d3mTestPlan.file("targetAssets", 55 * WAD);
-        vat.file("Line", 45 * RAD);
+        d3mTestPlan.file("targetAssets", 55 * WAD); // Increasing target in 5 WAD
+        vat.file("Line", vat.debt() - 5 * RAD);
         directDepositHub.exec(ilk);
 
         // Ensure we unwound our position to debt ceiling
         (uint256 ink, uint256 art) = vat.urns(ilk, address(d3mTestPool));
-        assertEq(ink, 45 * WAD);
+        assertEq(ink, 45 * WAD); // Instead of 5 WAD more results in 5 WAD less due debt ceiling
```

Comment:
Should we add 2 tests similar to the above for which a `wind` operation ends up being ceiled by line and Line?

Reply:
@talbaneth tests added.